Azure/azure-cli
# {Storage} Bump mgmt sdk to 17.1.0 and use api version 2021-02-01

**Description**

Bump version:
azure-mgmt-storage: 17.0.0 -> 17.1.0
api version: 2021-01-01 -> 2021-02-01

**Testing Guide**

`azdev test --no-exitfirst`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
